mediapipe/tasks/python/components/proto/segmenter_options.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Segmenter options data class."""
+
+import dataclasses
+import enum
+from typing import Any, Optional
+
+from mediapipe.tasks.cc.components.proto import segmenter_options_pb2
+from mediapipe.tasks.python.core.optional_dependencies import doc_controls
+
+_SegmenterOptionsProto = segmenter_options_pb2.SegmenterOptions
+
+
+class OutputType(enum.Enum):
+  UNSPECIFIED = 0
+  CATEGORY_MASK = 1
+  CONFIDENCE_MASK = 2
+
+
+class Activation(enum.Enum):
+  NONE = 0
+  SIGMOID = 1
+  SOFTMAX = 2
+
+
+@dataclasses.dataclass
+class SegmenterOptions:
```

Comment:
Please follow the same API pattern of the C++ API (https://github.com/google/mediapipe/blob/master/mediapipe/tasks/cc/vision/image_segmenter/image_segmenter.h#L36), and enumerate options of SegmenterOptions in ImageSegmenterOptions, instead of creating its own struct.

Reply:
@lu-wang-g I've changed the API so that it enumerates the options of `SegmenterOptions` in `ImageSegmenterOptions` now.